app/models/authority.rb

```diff
@@ -3,35 +3,48 @@ class Authority < ActiveFedora::Base
    extend ActiveTriples::Configurable
    include ActiveModel::Validations
 
-  property :label, predicate: ::RDF::Vocab::SKOS.prefLabel
-  property :alternate_label, predicate: ::RDF::Vocab::SKOS.altLabel
-  property :public_uri, predicate: ::RDF::Vocab::SKOS.exactMatch
+  property :label, predicate: ::RDF::Vocab::SKOS.prefLabel, multiple: false
+  property :alternate_label, predicate: ::RDF::Vocab::SKOS.altLabel, multiple: false
+  # xsd:anyURI
+  property :exact_match, predicate: ::RDF::Vocab::SKOS.exactMatch, multiple: true
 
   validates :label, presence: { message: 'Label is required.' }
   validates_with LabelExistsValidator
+  validates :exact_match, url: true, allow_blank: true
 
   def self.is_authority?(record)
-    record.is_a?(Authority) || record.class.ancestors.include?(Authority) || record.is_a?(Agent) || record.is_a?(Concept)
+    record.is_a?(Authority) || record.class.ancestors.include?(Authority) || record.is_a?(UcsdAgent) || record.is_a?(Concept)
```

Comment:
Yes, I think record.class.ancestors.include? should work and I can remove other explicit checks.